.github/workflows/pythonapp.yml

```diff
@@ -16,6 +16,19 @@ jobs:
       with:
         args: "src/ tests/ --check"
 
+  pretify-js:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Cloning the repository
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Prettify the JS Code
+      uses: creyD/prettier_action@v2.0
+      with:
+        prettier_options: 'prettier --check "src/server/static/**"'
```

Comment:
Same comment here, what about templates?

Reply:
On it

Reply:
Done in 4383116705921f0c4c3cc63023ca70e6a12fc2c1